parsec/backend/postgresql/user_queries/find.py

```diff
@@ -88,6 +108,32 @@ async def query_find(
     return results, len(all_results)
 
 
+@query()
+async def query_retrieve_human(
+    conn, organization_id: OrganizationID, arguments: List[Tuple[str, str, str]]
```

Comment:
This `arguments` param seems far to broad for it own good !

It would be better to have multiple query with each the exact parameters needed (this improve the readability of the caller code, simplify the implementation of this function and remove the need for the `_q_retrieve_human_factory` or at least for the need to call it dynamically with a lru cache !)